lib/__tests__/is-visible.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env jest */
+
+import TestAssertions from '../helpers/test-assertions';
+
+describe('assert.dom(...).isVisible()', () => {
```

Comment:
can we add a comment here explaining why this is difficult to test in the jsdom-based testing system and possibly a reference to the Ember-based tests?

Reply:
We can do that for sure.